### PR TITLE
feat: model-animations

### PR DESCRIPTION
**Pull request by AnkushS27**

### CODE REVIEW

File: src/components/section/heroSection.tsx [hunk 1]
```diff
@@ -1,30 +1,86 @@
...
+
+// import aos styles
+import 'aos/dist/aos.css';
+import { Akzidenzgroteskbe, FKScreamerBlack } from '@/fonts/fonts';
 
 export default function HeroSection() {
+  useEffect(() => {
+    AOS.init({
+      duration: 700,
+      once: false,
```
Comment (manavbabber) [inline src/components/section/heroSection.tsx:16]: Please check with once: true

File: src/helpers/handleCameraFOV.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    handleResize(); // Initial call
+    window.addEventListener('resize', handleResize);
+
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, [camera]);
+
+  useFrame(() => {
```
Comment (manavbabber) [inline src/helpers/handleCameraFOV.tsx:37]: may be useFrame is redundant as we are updating matrix in useEffect?

**Review by manavbabber — CHANGES_REQUESTED** [review]

File: src/components/scene/aeriusLogoScene.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        far: 1000,
+        near: 0.1,
+        fov: 32.269,
+        position: [-0.078, 0.935, 11.351],
+      }}
+    >
+      <color attach='background' args={['#000']} />
+      <directionalLight position={[-5, -5, 5]} intensity={4} />
+      <Environment preset='night' />
+      <Suspense fallback={null}>
```
[the remark this answers is not in the document]
Comment (AnkushS27) [reply src/components/scene/aeriusLogoScene.tsx:22]: Removed suspense from all models as not required now.

File: src/components/section/ourWritingsSection.tsx [hunk 4]
```diff
@@ -74,12 +83,15 @@ export default function OurWritingsSection() {
...
             </div>
-          </div>
+          </Link>
           {/* Bottom Line */}
           <div className='hidden h-[70%] min-h-[250px] w-[2px] bg-white/50 md:block'></div>
         </div>
-        <div className='sticky top-[990px] flex h-[50vh] flex-col items-center justify-center pb-5 md:top-52 md:mt-[160vh] md:h-screen'>
-          <div className='group flex cursor-pointer flex-col justify-between gap-20 border bg-black p-5 md:gap-40 md:bg-transparent'>
+        <div className='sticky top-[990px] z-30 flex h-[50vh] flex-col items-center justify-center pb-5 md:top-52 md:mt-[160vh] md:h-screen'>
+          <Link
```
[the remark this answers is not in the document]
Comment (AnkushS27) [reply src/components/section/ourWritingsSection.tsx:91]: This link will take you to the medium website where the blog is written.

File: src/helpers/handleCameraFOV.tsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+export default function AdjustCamera({
+  mobileFov,
+  desktopFov,
+  aspectRatio,
+}: AdjustCameraProps) {
+  const { camera } = useThree();
+
+  useEffect(() => {
+    const handleResize = () => {
```
[the remark this answers is not in the document]
Comment (AnkushS27) [reply src/helpers/handleCameraFOV.tsx:20]: done

File: src/components/section/heroSection.tsx (hunk 1 above)
Comment (AnkushS27) [reply src/components/section/heroSection.tsx:16]: done

File: src/helpers/handleCameraFOV.tsx (hunk 2 above)
Comment (AnkushS27) [reply src/helpers/handleCameraFOV.tsx:37]: Yes, removed the useFrame.

**Review by rahulghangas — APPROVED** [review]

**Pull request merged**